### PR TITLE
 feat：Add talk pages（#116）

### LINKED ISSUE #116: Feature: Add talk pages.
### Checklist

- [X] I've searched for similar issues and couldn't find anything matching
- [X] I've discussed this feature request in the [OpenIMSDK Slack](https://join.slack.com/t/openimsdk/shared_invite/zt-1tmoj26uf-_FDy3dowVHBiGvLk9e5Xkg) and got positive feedback

### Is this feature request related to a problem?

None

### Solution Description

Restore the design draft and add the customer service system dialog function.

### Benefits

Improve system functionality

### Potential Drawbacks

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by MC-kanon**
<br>

#### 👀 What this PR does / why we need it:

- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

#### 🅰 Which issue(s) this PR fixes:

Fixes #116

### CODE REVIEW

**Comment by IRONICBo** [comment]
@MC-kanon Please sign your CLA and recheck, with `I have read the CLA Document and I hereby sign the CLA` comment.

File: web/src/views/chat/index.vue [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+} from 'tdesign-icons-vue-next';
+import { ref, onMounted, reactive } from 'vue';
+import OpenKFLogo from '@/assets/openkf-logo.svg';
+import LoginBG from '@/assets/opekf-login-bg-light.png'; // change next time
+import Message from '@/components/Message.vue';
+import { getSDK } from '@/utils/open-im-sdk-wasm';
+const IMSDK = getSDK();
+
+const messageList = reactive<string[]>([
+    '1111111111111111111111111111111111111111111111111111111111111111111111111111111',
```
Comment (IRONICBo) [inline web/src/views/chat/index.vue:88]: It's your mock data?

File: web/src/views/chat/index.vue [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+    //     .then(({ data }) => {
+    //         // 调用成功
+    //         console.log('success', data);
+    //     })
+    //     .catch(({ errCode, errMsg }) => {
+    //         // 调用失败
+    //         console.log('error', errCode);
+    //     });
+    messageList.push(message.value);
+    console.log('发送了一条消息...');
```
Comment (IRONICBo) [inline web/src/views/chat/index.vue:111]: Use english please.

File: web/src/components/Message.vue [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<template>
+    <div :class="[isLeft ? 'left' : 'right']">
+        <t-avatar :image="avatarSrc" v-if="isLeft" size="large" />
```
Comment (IRONICBo) [inline web/src/components/Message.vue:3]: avatarSrc maybe empty, so you can update with character avatar.
```bash
        <template #avatar="{ row }">
          <t-avatar v-if="row.avatar === ''">{{ getAvatarString(row.nickname) }}</t-avatar>
          <t-avatar v-else :alt="getAvatarString(row.nickname)" hide-on-load-failed="true" :image="row.avatar" />
        </template>
```

**Review by IRONICBo — CHANGES_REQUESTED** [review]

File: web/src/views/chat/index.vue (hunk 1 above)
Comment (MC-kanon) [reply web/src/views/chat/index.vue:88]: temp data

File: web/src/views/chat/index.vue (hunk 2 above)
Comment (MC-kanon) [reply web/src/views/chat/index.vue:111]: ok

**Comment by MC-kanon** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by MC-kanon** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by cubxxw — APPROVED** [review]

**Review by IRONICBo — APPROVED** [review]

**Pull request merged**